src/Components/Header/Header.js

```diff
@@ -1,19 +1,52 @@
 import React, { Component } from 'react'
 import {NavLink} from 'react-router-dom'
 import '../Header/Header.scss'
-import {logo, searchIcon, contactIcon, cartIcon, arrowDown} from '../Icons/Icons'
+import {logo, searchIcon, contactIcon, cartIcon, arrowDown, moreOptions} from '../Icons/Icons'
+import SearchInput from '../SearchInput/SearchInput';
+import MobileButton from '../MobileButton/MobileButton';
+import Popup from '../Popup/Popup';
+
 
 export default class Header extends Component {
+    state = {
+        activatePopup: false,
+        setPopup: ''
+    }
+
+    handlePopupChange = (id) => {
+        this.setState({
+            activatePopup: true,
+            setPopup: id
```

Comment:
Naming pattern "setSOMETHING" "getSOMETHING" are commonly using for methods. 
For this var use could use "selectedPopupId"

Reply:
Done.